Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -1964,6 +1965,27 @@ - (void)setMinimumViewportInset:(CocoaEdgeInsets)minimumViewportInset maximumVie
     _maximumViewportInset = maximumViewportInset;
 }
 
+- (void)getApplicationManifestWithCompletionHandler:(void (^)(WKApplicationManifest *))completionHandler
+{
+    THROW_IF_SUSPENDED;
+#if ENABLE(APPLICATION_MANIFEST)
+    _page->getApplicationManifest([completionHandler = makeBlockPtr(completionHandler)](const std::optional<WebCore::ApplicationManifest>& manifest) {
+        if (completionHandler) {
+            if (manifest) {
+                auto apiManifest = API::ApplicationManifest::create(*manifest);
+                WKApplicationManifest *wkManifest = [[WKApplicationManifest alloc] initWithApplicationManifest:WTFMove(apiManifest)];
+                completionHandler(wkManifest);
+                [wkManifest release];
+            } else
+                completionHandler(nil);
```

Comment:
WKWebView.h declares this completion handler to take a non-null WKApplicationManifest (due to `NS_ASSUME_NONNULL`). Should it be made nullable if we are going to pass a `nil` in some cases?